core/server/adapters/scheduling/SchedulingDefault.js

```diff
@@ -147,30 +150,31 @@ SchedulingDefault.prototype._deleteJob = function (object) {
  * we don't want to use process.nextTick, this would block any I/O operation
  */
 SchedulingDefault.prototype._execute = function (jobs) {
-    var keys = Object.keys(jobs),
+    const keys = Object.keys(jobs),
         self = this;
```

Comment:
There are a couple of `self` usages in this file. Can they be removed when using arrow functions?

Reply:
I tried to completely refactor out self, but some integration tests started failing. Therefore, I left self in for now. I'd like to revisit it in the future, but I didn't want that to stop from me submitting what I had done so far.

Reply:
Ok 👍